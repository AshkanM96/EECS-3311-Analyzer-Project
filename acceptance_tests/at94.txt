sum
open_set_enumeration
integer_constant(3)
open_set_enumeration
integer_constant(4)
negative
subtract
integer_constant(6)   
integer_constant(5)
close_set_enumeration
integer_constant(6)
close_set_enumeration
type_check
evaluate
