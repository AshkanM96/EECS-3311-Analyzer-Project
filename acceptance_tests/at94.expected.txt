  Expression currently specified: ?
  Report: Expression is initialized.
->sum
  Expression currently specified: (+ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (+ {?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: (+ {3, ?})
  Report: OK.
->open_set_enumeration
  Expression currently specified: (+ {3, {?}})
  Report: OK.
->integer_constant(4)
  Expression currently specified: (+ {3, {4, ?}})
  Report: OK.
->negative
  Expression currently specified: (+ {3, {4, (- ?)}})
  Report: OK.
->subtract
  Expression currently specified: (+ {3, {4, (- (? - nil))}})
  Report: OK.
->integer_constant(6)
  Expression currently specified: (+ {3, {4, (- (6 - ?))}})
  Report: OK.
->integer_constant(5)
  Expression currently specified: (+ {3, {4, (- (6 - 5)), ?}})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (+ {3, {4, (- (6 - 5))}, ?})
  Report: OK.
->integer_constant(6)
  Expression currently specified: (+ {3, {4, (- (6 - 5))}, 6, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (+ {3, {4, (- (6 - 5))}, 6})
  Report: OK.
->type_check
  Expression currently specified: (+ {3, {4, (- (6 - 5))}, 6})
  Report: (+ {3, {4, (- (6 - 5))}, 6}) is not type-correct.
->evaluate
  Expression currently specified: (+ {3, {4, (- (6 - 5))}, 6})
  Report: Error (Expression is not type-correct).
